packages/office-ui-fabric-react/src/components/Rating/Rating.styles.ts

```diff
@@ -38,7 +38,7 @@ export function getStyles(props: IRatingStyleProps): IRatingStyles {
   const ratingVerticalPadding = 8;
   const ratingHorizontalPadding = 2;
 
-  const ratingStarUncheckedColor = palette.neutralTertiary;
+  const ratingStarUncheckedColor = palette.neutralSecondaryAlt;
```

Comment:
Definitely not use this color because there is an open issue for removing the use of this color: #6575